cmake/Install.cmake

```diff
@@ -147,6 +149,7 @@ install(DIRECTORY share/tools/web_config
         PATTERN "*.html"
         PATTERN "*.py"
         PATTERN "*.js"
+        PATTERN "*.theme"
```

Comment:
To answer the question in your commit message, yes, you need this - the `FILES_MATCHING` argument to `install(DIRECTORY)` disables the copying of any non-matching files (see https://cmake.org/cmake/help/latest/command/install.html)